evmrpc/simulate.go

```diff
@@ -389,9 +389,20 @@ func (b *Backend) RPCGasCap() uint64 { return b.config.GasCap }
 
 func (b *Backend) RPCEVMTimeout() time.Duration { return b.config.EVMTimeout }
 
+func (b *Backend) chainConfigForHeight(height int64) *params.ChainConfig {
+	ctx := b.ctxProvider(height)
+	ctx = ctx.WithIsTracing(true)
```

Comment:
Not sure if we should always forcing true here? I remember this sometimes cause perf issues

Reply:
@stevenlanders Any thoughts here? I remember we find out something around this earlier